README.md

```diff
@@ -62,7 +62,9 @@ Please file an issue: https://github.com/sourcegraph/sourcegraph-jetbrains/issue
 ## Development
 
 - Start IntelliJ and choose `Check out from Version Control` -> `Git` -> `https://github.com/sourcegraph/sourcegraph-jetbrains`
-- Develop as you would normally (hit Debug icon in top right of IntelliJ).
+- Develop as you would normally (hit Debug icon in top right of IntelliJ) or using gradlew commands:
+  1. `./gradlew runIde` to run an IDE instance with sourcegraph plugin installed
+  2. `./gradlew buildPlugin` to build plugin artifact (`build/distributions/Sourcegraph.zip`)
```

Comment:
Im curious if the line [here](https://github.com/sourcegraph/sourcegraph-jetbrains/pull/18/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R72) needs to be updated for the introduction of this new task and/or what the difference is between the two? I'm unfamiliar with Intellij plugin development as a whole :slightly_smiling_face:

Reply:
Hi @Strum355! `gradle runIde` launch an IDE instance with the installed plugin, which is used for local dev/test. You could also set up a Run/Debug configuration for it.
![image](https://user-images.githubusercontent.com/3115235/109817862-5d170000-7c6d-11eb-8328-bdf631f6b853.png)

`gradle buildPlugin` trigger the build process and generate a packaged jar/zip file which can be later published to the [IntelliJ plugin repository](https://plugins.jetbrains.com/) or installed to IDE by user from local disk. The plugin jar/zip file could be delivered to users for testing/dogfooding before released to plugin repository.

![image](https://user-images.githubusercontent.com/3115235/109818234-baab4c80-7c6d-11eb-8930-cba82f35392f.png)